processor/deltatocumulativeprocessor/go.mod

```diff
@@ -0,0 +1,43 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor
+
+go 1.20
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil => ../../pkg/pdatautil
```

Comment:
actually not, because components are their own modules depending on the parent module they need those replaces, otherwise MVS would try to download the parent module from the internet